ml-proto/host/import/env.ml

```diff
@@ -0,0 +1,39 @@
+open Values
+open Types
+
```

Comment:
Running compiled code as a stopgap until we have agreement on what libc should look like. When we have an agreed libc we'll likely do something like syscalls, and abort/exit will be simple renames. libc still won't be "standard", but it'll be an ad-hoc thing we end up using.

I'll be testing this on the waterfall, but something that only works on the waterfall is a silly idea so I want to avoid silly.